go/vt/sqlparser/sql.y

```diff
@@ -5673,39 +5677,11 @@ expression_list:
     $$ = append($1, $3)
   }
 
-value_expression:
+simple_value_expression:
```

Comment:
I don't think this new rule is pulling its weight. Just revert and use `value_expression` for AS OF, let something downstream of the parser fix any issue.

I say this because `value_expression` already has a bunch of non-value garbage in it, and this new name doesn't really fit. We'll fix later.